### DISCUSSION TITLE
Feature Flags / Disabling Pages

### DISCUSSION THREAD

**Discussion by chrisvander** [vikejs/vike#1497] [Discussion] [answered]
Hi,

I have a usecase where I need to disable certain features (and the associated Vike pages) in production, while I may still want those features on in development. Is there a way to disable a page completely based on an environment variable?

Thanks!

**Comment by brillout** [comment, marked as answer]
How about using a guard() hook?

**Reply by chrisvander** [reply]
Works well enough, thanks! I wonder if there's a way to opt it out of the build entirely, but this will work for now.

**Reply by brillout** [reply]
> opt it out of the build entirely

I think https://github.com/vikejs/vike/issues/341 will do the trick.
